### ISSUE TITLE
Built into exe with memory offset for latest version of Tekken 7

### ISSUE THREAD

**Issue by new-parad1gm** [roguelike2d/TekkenBot#54]
[TekkenBotPrime.zip](https://github.com/roguelike2d/TekkenBot/files/1361515/TekkenBotPrime.zip)

**Comment by yup-yup** [comment]
Input command column doesn't work for me ("N/A" in that field). Anyone else have the same issue? What could be the problem?

**Comment by MintPaw** [comment]
Yeah inputs says N/A, otherwise, everything in Frame Data mode seems to work.

**Comment by tempname2640** [comment]
I can't get it working. When I click the exe file I get an error "Windows cannot access the specified device, path or file. You may not have the appropriate permissions to access the item" then the exe file just disappears and I'm left with tekkenbot.exe.manifest Tried redownloading and retrying. Running as admin, extracting with 7zip instead of winrar. Never had any issues with the normal tekken bot updates. Do you have any ideas what could be wrong based on the error msg? I've talked to some people who used this update and they had no issues aside from inputs not showing.

**Comment by tempname2640** [comment]
Nvm figured it out looks like avast was blocking the exe file.

**Comment by TrickNachos** [comment]
Thank you so much!  My friends and I have all been waiting for this since it was broken.

Thanks new-parad1gm.

**Comment by smb3** [comment]
I have the N/A command issue as well.

**Comment by Pobega** [comment]
@new-parad1gm would you mind writing down exactly how you were able to generate the exe using Pyinstaller? I've tried doing it myself, but I always end up with an executable claiming it doesn't have a python36.dll

If you wanna discuss you can find me on Steam as Dr Pobega with the same avatar.

**Comment by new-parad1gm** [comment]
@Pobega just sent a FR on steam, its from cmdr faggenheimer.

**Comment by MintPaw** [comment]
@Pobega You're in build/TekkenBotPrime I assume? You should be in dist/TekkenBotPrime

**Comment by TrickNachos** [comment]
Yes I noticed a small increase in all file sizes was wondering if it was the debug version instead of a release.
